src/plugins/vis_type_timelion/server/series_functions/es/lib/build_request.js

```diff
@@ -27,7 +27,10 @@ export default function buildRequest(config, tlConfig, scriptedFields, timeout)
   bool.must.push(timeFilter);
 
   // Use the kibana filter bar filters
-  if (config.kibana) {
+  if (
+    config.kibana &&
+    !tlConfig.settings[UI_SETTINGS.COURIER_IGNORE_FILTER_IF_FIELD_NOT_IN_INDEX]
```

Comment:
@VladLasitsa @DziyanaDzeraviankina  could you please explain me how it should work? 
If I understand properly with that changes it's doesn't matter if fields exist in index pattern or not. We ignore filter based on `UI_SETTINGS.COURIER_IGNORE_FILTER_IF_FIELD_NOT_IN_INDEX`